shared/types/src/lib/config.types.ts

```diff
@@ -0,0 +1,99 @@
+/** ****************************************************************
+ * This contains a few types pulled from the JavaScript SDK
+ * TODO: Refactor the SDK to use these shared types instead of the internal ones
+ */
+import type { ConfigOptions } from '@forgerock/javascript-sdk';
+
+/** ****************************************************************
+ * NOTE: Pulled out of the SDK's /config/enums.ts file
+ */
+export enum ActionTypes {
```

Comment:
I'd favor a union over enum. its a slightly debated topic but most people say go with unions now.

Reply:
we can always refactor this when we do the `NOTE`

Reply:
Yeah, I'm with you about enums: I'm not a fan either. This was just copied from the original SDK code, and I didn't want to refactor it at the time. Once we get our vision with our next steps concerning "shared" code, we can address it then.